views/editor/projects.html

```diff
@@ -27,8 +27,10 @@
         {% endfor %}
         >
           <td class="project-title">
-            {{ project.title }}
-            <span class="project-information"></span>
+            <a target="_self" href="/{{locale}}/user/{{username}}/{{project.id}}/{{queryString}}"> 
```

Comment:
`queryString` here is not passed in by default. You need to explicitly pass it in here: https://github.com/mozilla/thimble.mozilla.org/blob/master/server/routes/projects/read.js#L76 only then can you use it in this link